stack/stack.rb

```diff
@@ -0,0 +1,35 @@
+class Stack
+  def initialize(size)
+    @size = size
+    @store = Array.new(@size)
+    @top = -1
+  end
+
+  def push(element)
+    if full?
```

Comment:
why don't we use guarded clause ``` raise "overflow error" if full? ```
Also, we raise error are we handling it?  we should build some kind of logger to handle and print information.